README.md

```diff
@@ -36,6 +36,13 @@ ANNOUNCEMENT_ID=[insert id here & remove brackets]
 VOICE_PARENT_ID=[insert id here & remove brackets]
 ```
 
+## Server setup
```

Comment:
Good call with this PR. I'd further, rather than recommending every user to create their own discord, also suggest adding a note to just have people who are interested join the official dev discord for this bot. Invites can be sent by DMing the current webmaster (who can also give you the test bot's env info)

Reply:
The final step in Setup already starts with "Reach out to the current tech head to be added to the test channel for the bot", so that should be covered! However, I can clarify that in the Server setup section if you think that'd be beneficial

Reply:
I think being added to the dev server/given access to the dev bot is the preferred path, but if thats covered then that works